projects/demo/src/modules/app/pages.ts

```diff
@@ -915,10 +915,10 @@ export const pages: TuiDocPages = [
         route: '/experimental/segmented',
     },
     {
-        section: 'Experimental',
+        section: 'Components',
         title: 'Surface',
         keywords: 'card, container, wrapper, image, blur, overlay',
-        route: '/experimental/surface',
+        route: '/components/surface',
```

Comment:
Should it be directive? Should button and link be directives, since they are actually directives? We need to think about it :)

Reply:
moved the discussion to the team chat